wallet/src/de/schildbach/wallet/ui/widget/ReceiveInfoView.kt

```diff
@@ -122,9 +122,9 @@ class ReceiveInfoView(context: Context, attrs: AttributeSet?) : ConstraintLayout
         try {
             val clipboardManager = context.getSystemService(Context.CLIPBOARD_SERVICE) as ClipboardManager
             if (amount != null) {
-                clipboardManager.primaryClip = ClipData.newPlainText("Dash payment request", paymentRequestUri)
+                clipboardManager.setPrimaryClip(ClipData.newPlainText("Dash payment request", paymentRequestUri))
             } else {
-                clipboardManager.primaryClip = ClipData.newPlainText("Dash address", address.toBase58())
+                clipboardManager.setPrimaryClip(ClipData.newPlainText("Dash address", address.toBase58()))
```

Comment:
Did something change with setPrimaryClip such that it cannot be implicitly called with `primaryClip = x`

Reply:
They added the `@NonNull` annotation to the ClipData parameter in ClipboardManager.setPrimaryClip(ClipData) method, whereas getPrimaryClip() is annotated as `@Nullable` and it cause Kotlin to fail with the use of property access syntax.
Here is the full answer: https://stackoverflow.com/a/57128936/795721